webhooks.md

```diff
@@ -325,12 +347,31 @@ ENABLE_WEBHOOKS=false GOWORK= OPERATOR_TEMPLATES=./templates make run
 
 Webhooks can be used outside of an OLM context if you want or need them.  However, using webhooks locally is non-trivial and is best handled by adding something like [this](https://github.com/openstack-k8s-operators/cinder-operator/pull/155/files) to the operator itself (as mentioned in the [Adding webhooks to an operator](#adding-webhooks-to-an-operator) section above).  This doc will only cover that use case for the time being.  Thus, do the following to use webhooks locally where the aforementioned support is present:
 
+**FR1 and earlier**
+
 1. First, if you installed the operator via OLM, remove its webhook definitions from its CSV:
 
 ```bash
 oc patch csv -n openstack-operators <your operator CSV> --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
 ```
 
+**FR2 and later**
+
+1. First, if this is a service operator (i.e. is not OpenStack operator) installed via OLM, remove its webhook definitions from its CSV:
+
+```bash
+oc patch csv -n openstack-operators <your operator CSV> --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
+```
```

Comment:
These commands are the same, but the reasoning for them is different between the RHOSO versions.